.gitignore

```diff
@@ -6,3 +6,4 @@ terraform.tfstate*
 /vendor
 /tests/rhcs_output
 terraform-provider-rhcs
+.idea
```

Comment:
Don't add IDE specific files to the repository git ignore
Each contributer/developer has the flexability to use his own IDE
You can add this to your local machine git ignore file
See for more info:
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Done.